awscli/customizations/scalarparse.py

```diff
@@ -36,13 +39,29 @@ def identity(x):
     return x
 
 
+def iso_format(value):
+    return parse_timestamp(value).isoformat()
+
+
+def choose_default_parsers(timestamp_format):
```

Comment:
I think the issue with that is the factory that botocore pulls the parsers from currently has to have its defaults set all at once with `set_parser_defaults` which means whatever is setting the timestamp parser also has to set the blob parser since we override that with `identity`. I didn't look too much into reworking the session's response parser factory.